doc/dynamo0p3.rst

```diff
@@ -1129,7 +1187,7 @@ inner_product
 
 Computes the inner product of the fields *field1* and *field2*, *i.e.*: ::
 
-  sumval = SUM(field1(:)*field2(:))
+   sumval = SUM(field1(:)*field2(:))
```

Comment:
There was a mix of 2 and 3 spaces in the text so I went for 3 without a particular reason, sorry! I can revert to 2 everywhere.